gdx/src/com/badlogic/gdx/scenes/scene2d/utils/TiledDrawable.java

```diff
@@ -78,5 +83,17 @@ public void draw (Batch batch, float x, float y, float width, float height) {
 				x += regionWidth;
 			}
 		}
+		batch.setColor(batchColor);
+	}
+
+	/** Creates a new drawable that renders the same as this drawable tinted the specified color. */
+	public TiledDrawable tintTiled (Color tint) {
+		TiledDrawable drawable = new TiledDrawable(this);
+		drawable.tint = tint.toFloatBits();
+		drawable.setLeftWidth(getLeftWidth());
+		drawable.setRightWidth(getRightWidth());
+		drawable.setTopHeight(getTopHeight());
+		drawable.setBottomHeight(getBottomHeight());
+		return drawable;

```

Comment:
This could be cleaner (an overload of `tint`) if this method of tinting was moved into the superclass. TextureRegionDrawable.tint could return a TextureRegionDrawable. But I didn't do that because I figured there was a good reason a SpriteDrawable is returned. But maybe that decision is a relic from when SpriteBatch didn't have `getPackedColor()` and `setColor(float)`.